### PR TITLE
fix(thing-model): move Thing Model code to td-tools, issue #680

### PR DESCRIPTION
**Pull request by lukesmolo**

### CODE REVIEW

File: packages/core/src/servient.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
```
Comment (danielpeintner) [inline packages/core/src/servient.ts:2]: I don't think this change is intended.

File: packages/td-tools/tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "resolveJsonModule": true,
```
Comment (danielpeintner) [inline packages/td-tools/tsconfig.json:4]: why is this needed?

**Comment by danielpeintner** [comment]
Thanks @lukesmolo  for your efforts. There seem to be some issues w.r.t. building that needs to be fixed.

**Comment by lukesmolo** [comment]
I will fix them as soon as possible.

File: packages/td-tools/tsconfig.json (hunk 2 above)
Comment (danielpeintner) [reply packages/td-tools/tsconfig.json:4]: I see

**Comment by danielpeintner** [comment]
Can you check the coding style issues also. The other failures seem to come from CI problems

**Comment by danielpeintner** [comment]
@relu91 @lukesmolo can you resolve the conflicts. Thanks!

**Review by relu91 — APPROVED** [review]

**Comment by relu91** [comment]
Todo: test thing-model-helpers in a browser

**Review by danielpeintner — APPROVED** [review]

**Comment by danielpeintner** [comment]
> Todo: test thing-model-helpers in a browser

We can do this at later point in time
